### DISCUSSION TITLE
I have an email address but can't verify it.

### DISCUSSION THREAD

**Discussion by kanhina1217** [community/community#102713] [GitHub Education]
### Hi everyone,

Hi everyone,
My application for GitHub Global Campus has been rejected.
The only reason given was “Attendees of this school must verify and use their academic email address.”

I have an academic email address. However, I cannot confirm this because my settings are such that I cannot verify emails. How do I get approved?

P.S.: I am using a translation app to write the text, so there may be mistakes. sorry.
![Screenshot 2024-01-30 11 31 20](https://github.com/community/community/assets/55534742/3e77cf98-672b-4397-a029-a8640bf6f60b)

**Comment by fury-05** [comment]
If you have academic email make sure you have added it to emails section in GitHub settings.

Now make it primary and then apply for GitHub education account using this email if prompted select academic email ( You can add many emails later and hide academy email later )

Even after adding email you will need to upload Student Id or Marksheet or Study Right Certificate using webcam.

If you have trouble uploading using webcam reply here will try to give its solution.

**Reply by kanhina1217** [reply]
Sorry, I guess I didn't get it across properly.
I can't receive emails so I can't add them to my primary email. (like the image below)

Below is the academic email.
![image](https://github.com/community/community/assets/55534742/b1ab576a-0f1e-4282-aa55-60c4cd0b2296)

**Reply by fury-05** [reply]
![image](https://github.com/community/community/assets/103818328/2619d131-56c6-4321-98f9-b2147cc55859)

You will receive an otp in this email address you need to confirm it.

Common trouble shooting for that check SPAM Folder.
Try sending email to this email from other email to check you are receiving emails or not ?

If not concern with your school / college IT Admin regarding incoming email policies.

**Comment by Mundednes** [comment]
After some digging (and a lot of trial and error), I found one that seemed legit. It scanned my inbox for any signs of unauthorized access and gave me a peace of mind. But here's the twist: not all [email hack checker](https://www.strongpasswordgenerator.org/email-hack-checker/) are created equal. Some are overly complex, while others are just scams in disguise. It's like navigating a minefield out there.

So, when I see discussions like this, I'm all ears. What's been your experience with email hack checkers? Have you found one that's trustworthy and actually works? I'm curious to hear your take on this. Let's swap stories and tips—it could save someone a major headache down the line.
